recipe/build.sh

```diff
@@ -1,11 +1,28 @@
 #!/usr/bin/bash
 
-set -x
+set -ex
 
-cp -r ${BUILD_PREFIX}/share/libtool/build-aux/config.* ./config
+cp -r ${BUILD_PREFIX}/share/libtool/build-aux/config.* ./build-aux
+
+if [[ "$target_platform" == linux* ]]; then
```

Comment:
What is this magic about?  I assume it is related to some tests failing on either Linux or OSX.  Some comment would be good here to clarify this

Reply:
this is to address failing regression tests on platform-specific builds